src/posthog-core.js

```diff
@@ -869,7 +869,11 @@ PostHogLib.prototype.identify = function (new_distinct_id, userPropertiesToSet,
         }
     }
 
-    this.reloadFeatureFlags()
+    // Reload active feature flags if the user identity changes.
+    // Note we don't reload this on property changes as these get processed async
```

Comment:
indeed property updates are async, so this stuff here was useless for property changes. wonder if we should poll for updates though to cover props changing